client/src/bundles/FieldEditor.js

```diff
@@ -95,7 +95,10 @@ jQuery.entwine('ss', ($) => {
           }
 
           $newField.addClass('flashBackground');
-          $('.cms-content-fields').scrollTop($('.cms-content-fields')[0].scrollHeight);
+          const $contenFields = $('.cms-content-fields');
```

Comment:
I think this class doesn't exist in the Elemental form. I'll see if I can find an equivalent class.